src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java

```diff
@@ -1407,4 +1414,34 @@ private void checkCreatingConsumer(String key, String value, HttpResponseStatus
             });
         consumer.get(TEST_TIMEOUT, TimeUnit.SECONDS);
     }
+
+    @Test
+    void createConsumerWithInvalidValueOfConfiguration(VertxTestContext context) throws InterruptedException, ExecutionException, TimeoutException {
```

Comment:
we have other methods checking for other wrong parameters, we should be more precise here so using something like `createConsumerWithInvalidFormat`, wdyt?

Reply:
make sense :+1: